extensions/wikia/CuratedContent/CuratedContentHelper.class.php

```diff
@@ -0,0 +1,160 @@
+<?php
+/**
+ * CuratedContent Helper
+ */
+class CuratedContentHelper {
+	const STR_ARTICLE = 'article';
+	const STR_BLOG = 'blog';
+	const STR_FILE = 'file';
+	const STR_CATEGORY = 'category';
+	const STR_EMPTY_CATEGORY = 'emptyCategory';
+
+	public function processLogic( $sections ) {
+		$processedSections = [ ];
+		foreach ( $sections as $section ) {
+			$processedSections[ ] = $this->processLogicForSection( $section );
+		}
+		return $processedSections;
+	}
+
+	private function processLogicForSection( $section ) {
+		$section['image_id'] = (int)$section['image_id'];
```

Comment:
`$section` comes straight from the request, right? What happens here if there is no `$section['image_id']` set?